ingest/genomes/genome_annotation_metadata.py

```diff
@@ -37,17 +33,17 @@ def upload_ensembl_gtf_products(ensembl_metadata, scp_species, config):
     remote_output_dir = config['remote_output_dir']
     remote_prod_dir = config['remote_prod_dir']
 
-    storage_client = get_gcs_storage_client(vault_path)
+    storage_client = utils.get_gcs_storage_client(vault_path)
 
     # FireCloud workspace for canonical SCP reference data:
     # https://portal.firecloud.org/#workspaces/single-cell-portal/scp-reference-data
     reference_data_bucket = gcs_bucket
-    target_folder = remote_output_dir
+    # target_folder = remote_output_dir
```

Comment:
either remove, or add comment explaining why this might need to be uncommented in the future

Reply:
Good point, I add a relevant TODO comment in 8a486fe.